lib/presentation/widgets/m3_widgets/cards/outlined_trip_card.dart

```diff
@@ -106,27 +108,28 @@ class OutlinedTripCard extends StatelessWidget {
               ],
             ),
           ),
-          const Divider(),
-          Padding(
-            padding: const EdgeInsets.symmetric(
-              horizontal: AppDimensions.d24,
-              vertical: AppDimensions.d10,
+          if (withActionButtons ?? true) const Divider(),
+          if (withActionButtons ?? false)
+            Padding(
+              padding: const EdgeInsets.only(
+                left: AppDimensions.d16,
+                right: AppDimensions.d16,
+                bottom: AppDimensions.d10,
+              ),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  WarningOutlinedButton(
+                    text: 'End the trip',
+                    onPressed: () {},
+                  ),
+                  PrimaryElevatedButton(
+                    text: 'Add expense',
+                    onPressed: () => context.router.push(CreateExpensesRoute(tripId: '', currency: '')),
+                  ),
+                ],
+              ),
```

Comment:
You created a widget for the buttons and you don't use it. Either delete the widget or use it here